src/components/reports/IndividualScoreReportTask.vue

```diff
@@ -0,0 +1,404 @@
+import { rawOnlyTasks, taskDisplayNames } from '@/helpers/reports';
+
+const props = defineProps({
+  studentData: {
+    type: Object,
+    required: true,
+  },
+  taskData: {
+    type: Object,
+    required: true,
+  },
+  rawTaskData: {
+    type: Object,
+    required: true,
+  },
+  expanded: {
+    type: Boolean,
+    required: false,
+  },
+});
+
+const extendedTaskData = {
+  ...props.taskData,
+  extendedTitle: {
+    swr: 'ROAR-Word',
+    'swr-es': 'ROAR-Palabra',
+    pa: 'ROAR-Phoneme',
+    sre: 'ROAR-Sentence',
+    vocab: 'ROAR-Picture Vocabulary ',
+    multichoice: 'ROAR-Multiple Choice',
+    morph: 'ROAR-Morphology',
+    cva: 'ROAR-Written Vocabulary',
+    letter: 'ROAR-Letter',
+    comp: 'ROAR-Comprehension',
+    phonics: 'ROAR-Phonics',
+    syntax: 'ROAR-Syntax',
+    fluency: 'ROAM-Fluency',
+  },
+  extendedName: {
+    swr: 'Single Word Recognition',
+    'swr-es': 'Single Word Recognition',
+    pa: 'Phonological Awareness',
+    sre: 'Sentence Reading Efficiency',
+    vocab: 'Picture Vocabulary',
+    multichoice: 'Multiple Choice Vocabulary',
+    morph: 'Morphological Awareness',
+    cva: 'Written Vocabulary',
+    letter: 'Letter Names and Sounds',
+    comp: 'Reading Comprehension',
+    phonics: 'Phonics',
+    syntax: 'Syntax',
+    fluency: 'Math Fluency',
+  },
+  extendedDescription: {
+    swr: 'This test measures your student’s skill in reading single words quickly and correctly',
+    'swr-es':
+      'This test measures how well a student can identify real words and made-up words. ' +
+      'The goal is for students to recognize words quickly and accurately, a skill called decoding. ' +
+      'High scores on this assessment indicate a readiness to be a skilled and fluent reader',
+    pa: 'This test measures how well your student can break down a spoken word into its individual sounds and choose or create a word with the same sounds',
+    sre: 'This test measures how quickly your student can silently read and understand sentences',
+    vocab: 'This test measures how well your student knows words by having them match a picture to a spoken word',
+    multichoice: 'Temporary description for multichoice',
+    morph:
+      'This test measures how well your student understands how parts of words, including prefixes and suffixes, can change the meaning of a word in a sentence',
+    cva: 'This test measures your students’ knowledge of words that are often used in the books they read at school',
+    letter:
+      'This test measures how well your student knows the names of letters and which letters are used to spell each sound',
+    comp: 'Temporary description for comp',
+    phonics:
+      'This test measures phonics knowledge by testing how well your student can match the sounds of a word to the spelling',
+    syntax: 'This test measures how well students understand sentences that vary from simple to complicated',
+    fluency: 'Temporary description for fluency',
+  },
+};
```

Comment:
I'd love to move these names and descriptions into the helpers/reports.js file so that they are commonly available to all of the report components/pages.